app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -12,17 +12,14 @@
 
 <%= render PageHeadingComponent.new.with_content(@presenter.heading) %>
 
-<p><%= @presenter.intro %></p>
-
-<%= render(AlertComponent.new) { t('two_factor_authentication.lockout_information') } %>
+<p class="maxw-mobile-lg margin-bottom-0"><%= @presenter.intro %></p>
```

Comment:
Curious about the max width: Was this prescribed in the design? I haven't seen it elsewhere in paragraph content, nor would I expect the paragraph to ever be wider than `30rem`.

Reply:
In mobile width, the alert and the updated intro content was larger than 30rem. The form has a max width of 30rem so they were not playing together nicely: 
![Screen Shot 2022-02-24 at 9 15 43 AM](https://user-images.githubusercontent.com/25465464/155541141-5f018759-8022-4dbf-a149-b5e8ffb5d575.png)

vs. the design:
![Screen Shot 2022-02-24 at 9 17 56 AM](https://user-images.githubusercontent.com/25465464/155541235-6d53ae2c-7dc2-47a8-b6d5-789701bce0d9.png)

Reply:
Gotcha, that's interesting. I didn't realize there was a max-width on the tile radios. Makes sense 👍